### PR TITLE
Ndrcn 116

### PR DESCRIPTION
**Pull request by somaraja**

### CODE REVIEW

**Review by simms11 — APPROVED** [review]

File: app/views/ImporterAddressConfirmationView.scala.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+@pageHint = {
+    <h2 class="govuk-heading-m" style="margin-bottom: 0.3em">
+        @messages("importerAddress.hint")
+        <p/>
+    </h2>
+}
+
+@h4_1 = {
+    <strongr >
+        NE98 1AE
```
Comment (lmugnai) [inline app/views/ImporterAddressConfirmationView.scala.html:39]: @somaraja this is the postcode that gets pulled through from the previous screen user answers. Can we just leave this blank for now as we haven't done the plumbing yet?

File: app/views/ImporterAddressConfirmationView.scala.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+    @components.heading("importerAddressConfirmation.heading")
+
+    @pageHint
+
+    @h4_1
+    @link("", messages("importerAddressConfirmation.link1"))
+    <p/>
+    @h4_2
+    @govukSelect(Select(
```
Comment (lmugnai) [inline app/views/ImporterAddressConfirmationView.scala.html:63]: @somaraja I think we can just leave the DDLB empty and populate it when we have the data.

File: app/views/ImporterAddressConfirmationView.scala.html (hunk 1 above)
Comment (somaraja) [reply app/views/ImporterAddressConfirmationView.scala.html:39]: @lmugnai As advised changes incorporated.

File: app/views/ImporterAddressConfirmationView.scala.html (hunk 2 above)
Comment (somaraja) [reply app/views/ImporterAddressConfirmationView.scala.html:63]: @lmugnai As advised changes incorporated.

**Pull request merged**